cmd/templates.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+func getUsageTemplate() string {
+	return `Usage:
+  {{.CommandPath}} [command]
+
+Commands:
+  {{rpad "setup" 29}} Setup guide to create your first feature flag
+  {{rpad "config" 29}} View and modify specific configuration values
+  {{rpad "completion" 29}} Generate the autocompletion script for the specified shell
```

Comment:
Dumb question, but I don't understand what this one means?

Will most ldcli users know what the autocompletion script does / why they'd want one? if yes, this is probably fine. if unsure, maybe something like "Generate a script to [some use/benefit] for the specified shell"

Reply:
yeah i think so - this is a built in command from the CLI library we're using

Reply:
sounds good, thanks